src/js/tabs/pid_tuning.js

```diff
@@ -2079,8 +2178,16 @@ TABS.pid_tuning.initialize = function (callback) {
         // update == save.
         $('a.update').click(function () {
             form_to_pid_and_rc();
-
+            self.configHasChanged = false;
             self.updating = true;
+
+            FC.ADVANCED_TUNING_ACTIVE.dMinRoll = FC.ADVANCED_TUNING.dMinRoll;
+            FC.ADVANCED_TUNING_ACTIVE.dMinPitch = FC.ADVANCED_TUNING.dMinPitch;
+            FC.ADVANCED_TUNING_ACTIVE.dMinYaw = FC.ADVANCED_TUNING.dMinYaw;
+
+            console.log('SAVE:', FC.ADVANCED_TUNING.dMinRoll, FC.ADVANCED_TUNING.dMinPitch, FC.ADVANCED_TUNING.dMinYaw);
```

Comment:
Do we need these?

Reply:
We are investigating some round errors in GUI. Will cleanup when finished.